content/using-sagemaker-managed-spot-training/notebooks/running_notebooks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Running Notebooks"
+chapter: false
+weight: 30
+---
+
+1. Within JupyterLab (Which you opened in the previous steps), navigate to the ***/ec2-spot-labs/ec2-spot-sagemaker-managed-spot-training/*** folder.
```

Comment:
are we sure that cloning the repo is the best approach here? there are tons of directories which are really not relevant and could appear overwhelming when first using JupyterHub. can the bootstrap simply delete these directories maybe?

Reply:
SageMaker currently doesn't support sparse checkouts, which would simplify this a bit, and I wanted to avoid creating another repo just for the notebooks (since they're already hosted in ec2-spot-labs). There are a couple other options I investigated (S3 hosted files, CodeCommit, Etc) but each adds some hosting complexity (need to manage a public S3 bucket, etc.). This ended up being by far the simplest route. If we get feedback that this is complex it's something we could change and likely the best path would be a new repo for the notebooks (ie: ec2-spot-notebooks) as that would minimize additional resources needed.